google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -266,13 +273,21 @@ private SpannerOptions(Builder builder) {
   private static class SpannerEnvironmentImpl implements SpannerEnvironment {
     private static final SpannerEnvironmentImpl INSTANCE = new SpannerEnvironmentImpl();
     private static final String SPANNER_OPTIMIZER_VERSION_ENV_VAR = "SPANNER_OPTIMIZER_VERSION";
+    private static final String SPANNER_OPTIMIZER_STATISTICS_PACKAGE_ENV_VAR =
+        "SPANNER_OPTIMIZER_STATISTICS_PACKAGE";
 
     private SpannerEnvironmentImpl() {}
 
     @Override
     public String getOptimizerVersion() {
       return MoreObjects.firstNonNull(System.getenv(SPANNER_OPTIMIZER_VERSION_ENV_VAR), "");
     }
+
+    @Override
```

Comment:
Used to set the optimizer statistics package through an environment variable, such as:
```
export SPANNER_OPTIMIZER_STATISTICS_PACKAGE="custom-package"
```